### ISSUE TITLE
Exception on RfcommDeviceService.FromIdAsync

### ISSUE THREAD

**Issue by pietrobr** [Azure-Samples/MyDriving#513]
On Win10 UWP I get an error on this line in ObdWrapper.cs
service = await RfcommDeviceService.FromIdAsync(device.Id)

device.id is "\\\\?\\BTHENUM#{00001101-0000-1000-8000-00805f9b34fb}_LOCALMFG&0002#8&861dc08&0&001DA568988B_C00000000#{b142fc3e-fa4e-460b-8abc-072b628b3c70}"

Error is "A method was called at an unexpected time. (Exception from HRESULT: 0x8000000E)"

Any idea?

**Comment by xiangyan99** [comment]
Did you debug into the app and see this? 
Could you check
1. Bluetooth is enabled and the obd device is paired and connected?
2. You give the permission of accessing bluetooth devices to the app?

Thank you.

**Comment by pietrobr** [comment]
After some investigation I found that the problem is in the first run (app not installed) the RfcommDeviceService.FromIdAsync(device.Id) try to present a message in the UI thread that causes the error. Try to uninstall and rerun the code to repro.

Probably the method "RequestAccessAsync" must me called in the onNaviagateTo of the UI. I didn't found it (???) .... to test I have just moved some code in the MyDrivingUWP project.
